CHANGES.md

```diff
@@ -44,6 +44,8 @@
 
   + Fix misalignment of cases in docked `function` match (#1498, @gpetiot)
 
+  + Preserve short-form extensions for structure item extensions (#1502, @gpetiot)
```

Comment:
I think an example would be useful here, for example that `open%ext M` will not get rewritten to `[%%ext open M]`.

Reply:
done